.changeset/famous-onions-jam.md

```diff
@@ -0,0 +1,9 @@
+---
+"saleor-app-taxes": minor
+---
+
+Set minimum Saleor version where app can be installed (3.10).
```

Comment:
@peelar I dont know the exact version that app is compatible with. Please confirm, I can change it